azavea/ashlar
# Release version 1.0.0 and set up deployments to PyPi from master

## Overview

This PR bumps Ashlar to version 1.0.0, updates the package metadata with more details, and sets up Travis to deploy to PyPi from commits to the `master` branch.

### Checklist


### Notes

Once this PR gets approved, I'll squash and merge and then do the following things to wrap up packaging:

- Merge `develop` into `master` to cut the 1.0.0 release to PyPi
- Tag this squashed commit with `v1.0.0`
- Continue developing on `develop`

## Testing Instructions

- Travis will run unit tests to confirm that the package is being installed correctly.
- When this PR gets pulled into `develop`, it will deploy the package to the PyPi test instance at https://test.pypi.org/project/ashlar/ to confirm that the package is being published properly. In a subsequent PR, I'll remove this deployment directive from Travis. Unfortunately, there doesn't seem to be a reliable way to test PyPi package CI from a PR, since encrypted variables are not available from pull requests in Travis; if you can think of a better way to test this, I'm all ears. 

Closes https://github.com/azavea/ashlar-2018-fellowship/issues/23.
